server/src-lib/Hasura/RQL/DDL/Schema/Table.hs

```diff
@@ -196,11 +199,19 @@ runSetExistingTableIsEnumQ (SetTableIsEnum tableName isEnum) = do
   buildSchemaCacheFor (MOTable tableName)
   return successMsg
 
+data SetTableConfiguration
+  = SetTableConfiguration
+  { _stcTable         :: !QualifiedTable
+  , _stcConfiguration :: !TableConfig
+  } deriving (Show, Eq, Lift)
+$(deriveJSON (aesonDrop 4 snakeCase) ''SetTableConfiguration)
+
 data SetTableCustomFields
   = SetTableCustomFields
   { _stcfTable             :: !QualifiedTable
   , _stcfCustomRootFields  :: !TableCustomRootFields
   , _stcfCustomColumnNames :: !CustomColumnNames
+  , _stcfIdentifier        :: !(Maybe G.Name)
```

Comment:
If `set_table_custom_fields` is now deprecated, should we just not allow a custom name to be given to this API? That way, people who want to use custom names will have a reason to migrate to the new API.

Reply:
Yes, it makes sense. I'll remove the custom name part from it.